great_expectations/rule_based_profiler/types/builder.py

```diff
@@ -38,7 +39,16 @@ def __init__(
             batch_request: specified in DomainBuilder configuration to get Batch objects for domain computation.
         """
         self._batch_list = batch_list
+
```

Comment:
should this constructor's type hints reference the base class?

Reply:
@donaldheppner Yes -- I think we can do an elegant change all around (in the next commit).  Thanks.